stix2elevator/convert_stix.py

```diff
@@ -65,8 +65,8 @@
 
 if stix.__version__ >= "1.2.0.0":
     from stix.report import Report
-if ((stix.__version__.startswith("1.1.1") and stix.__version__ > "1.1.1.6") or
-        (stix.__version__.startswith("1.2.0") and stix.__version__ > "1.2.0.2")):
+if (stix.__version__.startswith("1.1.1") or
+        stix.__version__.startswith("1.2.0")):
```

Comment:
Given that we already have stable versions with ais bindings. I think we can remove the if statement completely. Line 68-69.

Reply:
ok - I will do that.